content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst

```diff
@@ -1,262 +1,321 @@
+checking the box. Next, enter the desired number of calendar days. By configuring the security lead
+time, a buffer is set to account for potential delays in supplier deliveries.
+
+.. example::
+   Setting the :guilabel:`Security Lead Time for Purchase` to `2.00` days pushes back the
+   :guilabel:`Scheduled Date` of a receipt by a day. In that case, if a product
+   is initially scheduled to arrive on April 6th, with a two-day security lead time, the new
+   scheduled date for the receipt would be April 8th.
+
+   .. image:: scheduled_dates/vendor-security-LT.png
+      :align: center
+      :alt: Set security lead time for purchase from the Inventory > Configuration > Settings.
+
+.. _inventory/management/planning/scheduled/manuf-LT:
+
+Manufacturing - lead times
 ==========================
 
-Manufacturing Lead Time
+Streamline the procurement of manufactured products by following Odoo's recommended manufacturing
+order (MO) start dates based on the required date the product is needed in the warehouse. Determine
+the :abbr:`MO (Manufacturing Order)` deadline, which is the deadline to begin the manufacturing
+process to complete the product by the scheduled delivery date, by configuring manufacturing lead
+times and manufacturing security lead times.
+
+.. image:: scheduled_dates/manuf-lead-times.png
+   :align: center
+   :alt: Visualization of the determination of planned MO date manufacturing lead times.
+
+Manufacturing lead time
 -----------------------
 
-The *Manufacturing Lead Time* is the time needed to manufacture the
-product. To specify it, open the *Inventory* tab of your product form
-and add the number of days the manufacturing takes.
+Configure the manufacturing lead time directly on the product form by navigating to
+:menuselection:`Manufacturing app --> Products --> Products` and select the desired product. In the
+:guilabel:`Inventory` tab of the product, specify the calendar days needed to the manufacture the
+product in the field, :guilabel:`Manufacturing Lead Time`.
+
+.. example::
+   Specify a 14-day :guilabel:`Manufacturing Lead Time` for a product directly in the
+   :guilabel:`Inventory` tab of the product.
+
+   .. image:: scheduled_dates/set-manufacturing-LT.png
+      :align: center
+      :alt: View of the manufacturing lead time configuration from the product form
+
+Establish a :abbr:`MO (Manufacturing Order)` deadline based on the *expected delivery date*,
+indicated in the :guilabel:`Scheduled Date` field of the :abbr:`DO (Delivery Order)`. The :abbr:`MO
+(Manufacturing Order)` deadline, which is the :guilabel:`Scheduled Date` field on the **:abbr:`MO
+(Manufacturing Order)`**, is calculated as the *expected delivery date* - manufacturing lead time.
+This ensures that the manufacturing process begins on time to meet the delivery date. However it's
+important to note that lead times are based on calendar days. Lead times do not consider weekends,
+holidays, or work enter capacity.
```

Comment:
```suggestion
holidays, or work center capacity.
```

Reply:
A `dfn` tag could be fun here since people might be like "what is work center capacity?"! Example:

...holidays, or work center capacity. (:dfn:`the number of operations that can be performed at the work center simultaneously`)

Reply:
This is a great idea! I just don't like how the definition tag renders on the website. (This is my very irrational hill I'll die on that I will not force on others LOL)

```suggestion
do not consider weekends, holidays, or :abbr:`work center capacity (the number of operations that
can be performed at the work center simultaneously)`.
```
Instead, I added (albeit incorrectly) an abbreviation tag because it looks more like what I think the definition tag should look while rendered on the webpage. I'll defer to @StraubCreative for the final decision.

Reply:
hmm...think you'd want to use the `dfn` tag. Here's why:

if we use the `:abbr:` tag here subjectively, instead of the correct tag, it may lead to a bunch of whack-a-mole scenarios that will accumulate over time, which we'd have to fix later. It will also cause confusion among contributors who rely on the style guide and might invite unnecessary creativity around tag use.

@samueljlieber is looking into options around how this tag is rendered in the HTML, because yea, we do agree it could be improved.